tests/stdout/derive_mac/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "derive_mac"
+version = "0.0.0"
+edition = "2018"
+
+[lib]
+proc-macro = true
+
+[dependencies]
+syn = "0.15"
+quote = "0.6"
```

Comment:
Please make sure every line ends with a newline. Without that, this isn't considered a [text file](https://unix.stackexchange.com/a/18789/179081) and it messes up line-oriented tools. You should set your editor to always do this.